site/en/blog/new-in-chrome-87/index.md

```diff
@@ -0,0 +1,243 @@
+to the camera, but also to PTZ functionality.
+
+To request permission for PTZ functionality, call
+`navigator.mediaDevices.getUserMedia()` with the PTZ constraints. This will
+prompt the user to grant both regular camera and camera with PTZ permissions.
+
+```js
+try {
+  const opts = {video: {pan: true, tilt: true, zoom: true}};
+  const stream = await navigator.mediaDevices.getUserMedia(opts);
+  document.querySelector("#video").srcObject = stream;
+} catch (error) {
+  // User denies prompt, or
+  // matching media is not available.
+}
+```
+
+Finally, a call to `MediaStreamTrack.getSettings()` will tell you what the
+_camera_ supports.
+
+```js
+const [videoTrack] = stream.getVideoTracks();
+const capabilities = videoTrack.getCapabilities();
+const settings = videoTrack.getSettings();
+
+if ('pan' in settings) {
+  enablePan(capabilities, settings);
+}
+// Similar for tilt and zoom...
+```
+
+Once the user has granted permission, you can then call
+`videoTrack.applyConstraints()` to adjust the pan, tilt, and zoom.
+
+```js
+function enablePan(capabilities, settings) {
+  const input = document.getElementById('rangePan');
+  input.min = capabilities.pan.min;
+  input.max = capabilities.pan.max;
+  input.step = capabilities.pan.step;
+  input.value = settings.pan;
+
+  input.addEventListener('input', async () => {
+    const opts = { advanced: [{ pan: input.value }] };
+    await videoTrack.applyConstraints(opts);
+  });
+}
+```
+
+Personally, I’m really excited about PTZ, so I can hide my messy kitchen, but
+you'll have to check out the video to see that!
+
+Francois has a great post [Control camera pan, tilt, and zoom][wd-ptz] on
+web.dev with code samples, complete details the best way to request
+permission, and a demo, so that you can try it out, and see if your webcam
+supports PTZ.
+
+## Range requests and service workers {: #rr }
+
+[HTTP range requests][mdn-rr], which have been available in major browsers
+for several years, allow servers to send requested data to the client in chunks.
+This is especially useful for large media files, where the user experience is
+improved through smoother playback, enhanced scrubbing, and better pause and
+resume functions.
+
+Historically, range requests and services workers did not work well together,
+forcing developers to build work-arounds. Starting in Chrome 87, passing
+range requests through to the network from inside a service worker will
+"just work."
+
+```js
+self.addEventListener('fetch', (event) => {
+  // The Range: header will pass through
+  // in browsers that behave correctly.
+  event.respondWith(fetch(event.request));
+});
+```
+
+For an explanation of the issues with range requests and what's changed in
+Chrome 87, see Jeff’s article [Handling range requests in a service worker][wd-rr]
+on web.dev.
+
+## Origin Trial: Font access API {: #font-access }
+
+{% img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/x5qv7Ylxb9i2mtWkBwVt.png", alt="Screen shot of Photopea image editor" %}
```

Comment:
How do I get this to appear half-width, right side? I tried `className="w-figure--inline-right"` but that didn't work.